app/frontend/src/utils/transformUtils.js

```diff
@@ -64,3 +65,147 @@ export function attachAttributesToLinks(formSchemaComponents) {
     }
   });
 }
+
+
+/**
+ * This function could be removed from frontend in the next commits
+ * @function isEligibleLateSubmission
+ * Get All possible dates in given period with Term and Interval
+ *
+ * @param {Object[]} date An object of Moment JS date
```

Comment:
Can all of these parameter descriptions be more descriptive? It's a moment js date but what is it used for? The only way to understand is to read through the code because the parameter description doesn't say.